www/artists/jcowin/js/slideshow.js

```diff
@@ -0,0 +1,28 @@
+var slideIndex = 1;
+showSlides(slideIndex);
```

Comment:
A function shouldn't be used before it's defined. I recommend moving the function definition on line 14 to the top of the file, rather than the bottom.

Reply:
This is is the same exact code on the website that I linked. Brett said that this issue is resolved.
https://www.w3schools.com/howto/howto_js_slideshow.asp